irksome/stage.py

```diff
@@ -340,9 +346,11 @@ def __init__(self, F, butcher_tableau, t, dt, u0, bcs=None,
             self.update_problem,
             solver_parameters=update_solver_parameters)
 
-        self._update = self._update_general
+        if stiff_acc_huh:
+            self._update = self._update_riia
```

Comment:
Should we rename `_update_riia` as `_update_stiff_acc`?

Reply:
Good idea.